Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreter.cs

```diff
@@ -264,8 +297,8 @@ class AstPythonInterpreter : IPythonInterpreter, IModuleContext, ICanFindModuleM
         }
 
         public IEnumerable<string> GetModulesNamed(string name) {
-            var usp = GetUserSearchPathPackagesAsync().WaitAndUnwrapExceptions();
-            var ssp = _factory.GetImportableModulesAsync().WaitAndUnwrapExceptions();
+            var usp = GetUserSearchPathPackagesAsync(CancellationToken.None).WaitAndUnwrapExceptions();
+            var ssp = _factory.GetImportableModulesAsync(CancellationToken.None).WaitAndUnwrapExceptions();
```

Comment:
`GetUserSearchPathPackagesAsync` calls `GetImportableModulesAsync`. Should we change the order?

Reply:
It calls it with different arguments. The `AstPythonInterpreter` implementation of `GetUserSearchPathPackagesAsync` uses the static `GetImportableModulesAsync` to search its own search paths, while the `AstPythonInterpreterFactory.GetImportableModulesAsync` non-static version uses the factory's search paths.

Reply:
Ok